std_msgs/Header header
sensor_msgs/Image front_camera
sensor_msgs/Image internal_camera  

geometry_msgs/PointStamped gaze_position_3d

